packages/paste-website/src/pages/blog/2023-05-16-paste-newsletter.mdx

```diff
@@ -0,0 +1,155 @@
+export const meta = {
```

Comment:
chore: The file name needs to change to link appropriately from this page: https://deploy-preview-3270--paste-docs.netlify.app/blog